sql/src/main/java/io/crate/action/sql/SessionContext.java

```diff
@@ -73,8 +83,8 @@ public String defaultSchema() {
         return defaultSchema;
     }
 
-    public void setDefaultSchema(@Nullable String schema) {
-        defaultSchema = MoreObjects.firstNonNull(schema, Schemas.DEFAULT_SCHEMA_NAME);
+    public void setDefaultSchema(String schema) {
+        defaultSchema = Preconditions.checkNotNull(schema);
```

Comment:
Could use JDKs `requireNonNull` and add a message so that we don't get an empty NullPointerException